Scripts/Python/psnlYeeshaPageChanges.py

```diff
@@ -61,9 +62,9 @@
 
 respEnable = ptAttribResponder(5, "Enabled resp (if necessary)")
 respDisable = ptAttribResponder(6, "Disabled resp (if necessary)")
+bushDistrib = ptAttribClusterList(7, "Bush distributor")
```

Comment:
I thought this was the blue flowers, not the bushes?

Reply:
thats what Cyan named it in UruCC, probably better to rename it cluster components

Reply:
Pushed a change with better naming